### PR TITLE
DUOS-1997[risk=no] Updated Open and Close Collection functions for Admin and Chair Consoles

### PR DESCRIPTION
**Pull request by JVThomas**
Addresses [DUOS-1997](https://broadworkbench.atlassian.net/browse/DUOS-1997)

PR updates the open and cancel functions on the Admin and Chair console pages to handle the new `DarCollectionSummary` model format. 

PR has a dependency on the endpoint to be defined in https://github.com/DataBiosphere/consent/pull/1669

Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/pages/AdminManageDarCollections.js [hunk 1]
```diff
@@ -38,8 +38,8 @@ export default function AdminManageDarCollections() {
   }, []);
 
   const updateCollections = updateCollectionFn({collections, filterFn, searchRef, setCollections, setFilteredList});
-  const cancelCollection = cancelCollectionFn({updateCollections, role: 'admin'});
-  const openCollection = openCollectionFn({updateCollections});
+  const cancelCollection = cancelCollectionFn({updateCollections, role: 'Admin'});
+  const openCollection = openCollectionFn({updateCollections, role: 'Admin'});
```
Comment (shaemarks) [inline src/pages/AdminManageDarCollections.js]: nit: `USER_ROLES.admin` from `utils.js` might be a useful replacement for `role: 'Admin'`

File: src/pages/ChairConsole.js [hunk 2]
```diff
@@ -44,8 +44,8 @@ export default function ChairConsole(props) {
   }, []);
 
   const updateCollections = updateCollectionFn({collections, filterFn, searchRef, setCollections, setFilteredList});
-  const cancelCollection = cancelCollectionFn({updateCollections, role: 'chairperson'});
-  const openCollection = openCollectionFn({updateCollections});
+  const cancelCollection = cancelCollectionFn({updateCollections, role: 'Chairperson'});
+  const openCollection = openCollectionFn({updateCollections, role: 'Chairperson'});
```
Comment (shaemarks) [inline src/pages/ChairConsole.js]: similar nit as above

**Review by shaemarks — APPROVED** [review]
Tested locally with [DataBiosphere/consent#1669](https://github.com/DataBiosphere/consent/pull/1669) as the backend, and the dar collection table updated correctly after open and close collection actions for admins and chairs

**Review by connorlbark — APPROVED** [review]
looks good!

**Pull request merged**